src/common/exif.cc

```diff
@@ -1237,10 +1240,13 @@ static bool _exif_decode_exif_data(dt_image_t *img, Exiv2::ExifData &exifData)
             mat3mul(img->d65_color_matrix, correctmat[0], colmatrix[sel_illu]);
             break;
           case DT_LS_D55:
+          case DT_LS_Daylight:
+          case DT_LS_FineWeather:
```

Comment:
Do we just assume that FineWeather is the same as Daylight (then why it was introduced at all?) or is there some reference? I couldn't find it.

Reply:
This comes from the DNG SDK.

Reply:
Do you happen to have a link where exactly in the sources it is defined?

Reply:
Found it, nevermind, thanx.

Reply:
The problem with DNG SDK as a reference is that they also define StandardLightB as D55, which is quite off actually (4874). But I think we're indeed good with thinking that FineWeather means Daylight.